src/Compilers/VisualBasic/Portable/Binding/MemberSemanticModel.vb

```diff
@@ -794,6 +794,51 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         Friend Overrides Function GetOperationWorker(node As VisualBasicSyntaxNode, options As GetOperationOptions, cancellationToken As CancellationToken) As IOperation
+            Dim argumentNode = TryCast(node, ArgumentSyntax)
+            If argumentNode IsNot Nothing Then
```

Comment:
Vb And C# behave differently here.  In C# asking for the BoundNode for an argument gives you the bound node from the containing invocation containing the argument.  However, in VB asking for the BoundNode for an argument just returns null.  We have to have different code in each language to deal with the discrepancies.